ci/test_ubuntu.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+#
+# Test NodeJS for Ubuntu
+#
+set -o pipefail
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source $THIS_DIR/_init.sh
+source $THIS_DIR/scripts/set_git_info.sh
+
+export WORKSPACE=$GITHUB_WORKSPACE
+export SOURCE_ROOT=$GITHUB_WORKSPACE
+export SHOULD_GENERATE_COVERAGE_REPORT=1
```

Comment:
when it's 1 - should we gathered results and send them to codecov?

Reply:
Yes we should.